code/game/gamemodes/miniantags/slaughter/slaughter.dm

```diff
@@ -29,7 +29,7 @@
 	maxHealth = 200
 	health = 200
 	environment_smash = 1
-	//universal_understand = 1
+	//universal_understand = TRUE
```

Comment:
Can you remove this commented out line instead?

Reply:
Done